doc/Tex/RAM_SCB_run.tex

```diff
@@ -139,115 +82,48 @@
-change parameters at certain parts of a storm, or perhaps a simulation was 
-interrupted undesireably.  Restart files allow you to continue a simulation
-seamlessly.
-
-Throughout a simulation, restart files are being written to the {\tt restartOUT}
-folder of the run directory.  The files are written at a set frequency and at 
-the end of a successful run (both of these options are configuarble through
-the {\tt PARAM} interface.)  Only the most recent restart is saved, users
-can periodically pull these files out of {\tt restartOUT} if they prefer a 
-back log of restarts.  These files contain the full distribution functions as
-well as the start time, current time, and current iteration at the point that
-the restart was written.
-
-To restart a run, first move restart files from {\tt restartOUT} to 
-{\tt restartIN}. Then, either create a new {\tt PARAM.in} file or edit the
-existing one such that no \verb*l#STARTTIMEl command is present (remember, the
-restart file knows this information already), the run time exceeds the current
-time listed in the restart file, and the command \verb*l#RESTARTl is included.
-This tells RAM-SCB to restart the simulation rather than start anew.  Restarts
-are carefully implemented to preserve files in the output directories and 
-continue the simulation as if there was no interruption.
-
-For an example of restarting a simulation, run Test 2 and inspect the 
-{\tt PARAM.in} files as well as the contents of {\tt restartIN} and 
-{\tt restartOUT}.
+It is often desirable to split a simulation up into several parts. For example, you may want to split a very long simulation up into separate executions, or change parameters at certain parts of a storm, or perhaps a simulation was interrupted undesireably. Restart files allow you to continue a simulation seamlessly.
+
+Throughout a simulation, restart files are being written to the {\tt restartOUT} folder of the run directory. The files are written at a set frequency and at the end of a successful run (both of these options are configuarble through the {\tt PARAM} interface.) Only the most recent restart is saved, users can periodically pull these files out of {\tt restartOUT} if they prefer a back log of restarts. These files contain the full distribution functions as well as the start time, current time, and current iteration at the point that the restart was written.
+
+To restart a run, first move restart files from {\tt restartOUT} to {\tt restartIN}. Then, either create a new {\tt PARAM.in} file or edit the existing one such that no \verb*l#STARTTIMEl command is present (remember, the restart file knows this information already), the run time exceeds the current time listed in the restart file, and the command \verb*l#RESTARTl is included. This tells RAM-SCB to restart the simulation rather than start anew. Restarts are carefully implemented to preserve files in the output directories and continue the simulation as if there was no interruption.

```

Comment:
Is this still required? I thought we'd changed this at some point to have the latest restart linked/copied over to the restartIN directory automatically... I may well be misremembering this.

Reply:
That was the plan I believe, it doesn't currently happen though. But it should be a simple enough fix

Reply:
Fair enough. Let's keep this for now and then when the update gets made to the code the docs can be updated.